bot/seasons/valentines/savethedate.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+
+log = logging.getLogger(__name__)
+
+emoji = [":heart:", ":couple_with_heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+
+
+class SaveTheDate:
+    """
+    A cog to change a invokers nickname to a spooky one!
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def savethedate(self, ctx):
+        with open(Path('bot', 'resources', 'valentines', 'date_ideas.json'), 'r', encoding="utf8") as f:
+            data = load(f)
+            date_idea = random.choice(data['ideas'])
+            emoji_1 = random.choice(emoji)
+            emoji_2 = random.choice(emoji)
+            embed = discord.Embed(title=date_idea['name'],
+                                  description=f"{emoji_1}{date_idea['description']}{emoji_2}",
+                                  colour=0x01d277)
```

Comment:
[`bot.constants.Colours`](https://github.com/python-discord/seasonalbot/blob/master/bot/constants.py#L66) has a color for you to use.

Reply:
oki

Reply:
The colour @sco1 recommended was the pink one. It wasn't entirely clear, but I agree the embed probably should be pink to match valentines.

Reply:
okay